index.html

```diff
@@ -2119,17 +2119,11 @@ <h2>Interaction Model</h2>
         <h3>Properties</h3>
         <p>
           A Property is an Interaction Affordance that exposes the state of the Thing.
-          <span class="rfc2119-assertion" id="arch-property-readable">
-            The state exposed by a Property MUST be retrievable (readable).
-          </span>
-          <span class="rfc2119-assertion" id="arch-property-writable">
-            Optionally, the state exposed by a Property MAY be updated (writeable).
-          </span>
-          <span class="rfc2119-assertion" id="arch-property-observable">
-            <a>Things</a> MAY choose to make Properties observable by pushing the new state after a change
+          The state exposed by a Property can be retrievable (readable).
+          Optionally, the state exposed by a Property may be updated (writeable).
+           <a>Things</a> MAY choose to make Properties observable by pushing the new state after a change
```

Comment:
Since this is not an RFC assertion, please use a lower case "may"

Reply:
done